src/components/progress-bar/progrss-bar.spec.ts

```diff
@@ -32,4 +32,26 @@ test.describe("ProgressBar", () => {
     expect(headSegmentSize).toBe(expectedHeadSegmentSize);
     expect(completedSegmentSize).toBe(expectedCompletedHeadSize);
   });
+
+  test("should use panel-color-dark for background to provide contrast", async ({ fixture }) => {
```

Comment:
We have a helper for this. And I usually don't write tests for simple style changes unless I'm running something like an ARIA contrast test (which might be kind of cool)